testcases/open_posix_testsuite/conformance/interfaces/free/coverage.txt

```diff
@@ -0,0 +1,8 @@
+This file defines the coverage for the free() function testing.
+
+	Assertion	Status
+	1			YES
+	2			NO *
+
+	* - Hard to test
```

Comment:
Well you can at least do a test that calls free(NULL); exit(0); which does not cover much, but is still better than nothing.

Reply:
okie, added test in new patch.